Gemfile

```diff
@@ -39,7 +39,7 @@ gem 'hydra-role-management', '~> 0.2.0'
 gem 'rsolr', '~> 1.1.0'
 gem 'devise', '~> 3.0'
 gem 'devise-guests', '~> 0.3'
-gem 'iiif-presentation'
+gem 'iiif-presentation', github: 'tpendragon/osullivan', branch: 'no_singleton_classes'
```

Comment:
OK, I'm fine merging, just didn't want to go ahead if the osullivan PR was going to be merged.